src/goto-instrument/contracts/contracts.h

```diff
@@ -177,43 +175,38 @@ class code_contractst
     goto_programt::instructionst::iterator &ins_it,
     goto_programt &program,
     exprt &assigns,
-    const irep_idt &function_id,
     std::set<irep_idt> &freely_assignable_symbols,
     assigns_clauset &assigns_clause);
 
-  /// Creates a local variable declaration for each expression in operands,
```

Comment:
This doesn't fit with the commit's message.  It really isn't ensuring a consistent ordering of parameters.

Reply:
Good point, Martin! I broke down the commit into three. I'll keep this in mind for future PRs.